models/topics.js

```diff
@@ -54,4 +55,31 @@ exports.selectCommentsByArticleId = (article_id) => {
     });
 };
 
+exports.addComment = (article_id, username, body) => {
+    if (!username || !body) {
+        return Promise.reject({ status: 400, msg: 'missing object properties' })
+    } else {
+        return db
+        .query(`INSERT INTO users (username, name) VALUES ($1, $2) RETURNING*;`, [username, 'name'])
```

Comment:
I understand the logic behind adding a user, however, this looks like it will always add a new user if they don't exist. Ideally, we should instead be checking if the provided user does exist in the db, and if not, throw an error 🙂